specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,431 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2020-09-01",
+    "description": "To manage and control access to your resources, you can define customized policies and assign them at a scope."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/dataPolicyManifests/{policyMode}": {
+      "get": {
+        "tags": [
+          "DataPolicyManifests"
+        ],
+        "operationId": "DataPolicyManifests_GetByPolicyMode",
+        "summary": "Retrieves a data policy manifest.",
+        "description": "This operation retrieves the data policy manifest with the given policy mode.",
+        "x-ms-examples": {
+          "Retrieve a data policy manifest by policy mode": {
+            "$ref": "./examples/getDataPolicyManifest.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyMode",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy mode of the data policy manifest to get."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about a data policy manifest.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifest"
+            }
+          },
```

Comment:
mention 404 explicitly too in case of an unknown mode

Reply:
Removed schema from 404 response as I couldn't find any example for it (I would also have to specify a body for the response in the example)